File: src/files/RemovePunctuation.cpp
#include <iostream>
using namespace std;

int main()
{
    std::string str = "Welcome???@@##$ to#$% ProGrammar";
    for (int i = 0, len = str.size(); i < len; i++)
    {     
        if (ispunct(str[i]))
        {
            str.erase(i--, 1);
            len = str.size();
        }
    }
    std::cout << str;
    return 0;
}

File: src/files/RemovePunctuation.py
def Punctuation(string):
	punctuations = '''!()-[]{};:'"\,<>./?@#$%^&*_~'''
	for x in string.lower():
		if x in punctuations:
			string = string.replace(x, "")
	print(string)
string = "Welcome???@@##$ to#$% ProGrammar"
Punctuation(string)